addons/editor-devtools/blockly/Utils.js

```diff
@@ -56,9 +59,8 @@ export default class Utils {
   /**
    * Based on wksp.centerOnBlock(li.data.labelID);
    * @param blockOrId {Blockly.Block|{id}|BlockInstance} A Blockly Block, a block id, or a BlockInstance
-   * @param [force] {boolean} if true, the view always moves, otherwise only move if the selected element is not entirely visible
```

Comment:
Note: this was removed because it was unused
@TheColaber Correct?

Reply:
yea